### DISCUSSION TITLE
Use JSON file for request body

### DISCUSSION THREAD

**Discussion by bombillazo** [thunderclient/thunder-client-support#855] [Ideas]
Hello, great library, thanks for developing.

Is there a way to select a JSON file as the source of a JSON body? I find myself constantly copy pasting json files and it would be great to easily switch the file to use as the JSON body instead.

**Comment by rangav** [comment]
@bombillazo thanks for the feedback.

It’s already on roadmap.
https://github.com/rangav/thunder-client-support/issues/296

Please express your interest in the above link.
